fluentd/patch/logger/supervisor.rb

```diff
@@ -56,9 +56,9 @@ def initialize(path, level, chuser, chgroup, opts)
         $openShiftLogger = nil
         if !envPath.nil?
           path = envPath
-          age = Integer(ENV["LOGGING_FILE_AGE"])
-          size = Integer(ENV["LOGGING_FILE_SIZE"])
-          $openShiftLogger = Logger.new(path, age, size)
+          age = Integer(ENV["LOGGING_FILE_AGE"]) || 10
```

Comment:
You can use `irb` to test out small pieces of ruby code:
```
irb
irb(main):005:0> Integer(nil)
TypeError: can't convert nil into Integer
        from (irb):5:in `Integer'
        from (irb):5
        from /usr/bin/irb:12:in `<main>'
irb(main):006:0> Integer(ENV['FOO'])
TypeError: can't convert nil into Integer
        from (irb):6:in `Integer'
        from (irb):6
        from /usr/bin/irb:12:in `<main>'
irb(main):007:0> Integer(ENV['FOO'] || 10)
=> 10
```
so you need to use
```
          age = Integer(ENV["LOGGING_FILE_AGE"] || 10)
```
to avoid a possible exception

Reply:
or `(ENV["LOGGING_FILE_AGE"] || 10).to_i`